src/libraries/Common/src/Interop/Unix/System.Native/Interop.UnixFileSystemTypes.cs

```diff
@@ -106,20 +107,20 @@ internal enum UnixFileSystemTypes : uint
             panfs = 0xAAD7AAEA,
             pipefs = 0x50495045,
             proc = 0x9FA0,
-            pstorefs = 0x6165676C,
+            pstore = 0x6165676C,
             qnx4 = 0x002F,
             qnx6 = 0x68191122,
             ramfs = 0x858458F6,
             reiserfs = 0x52654973,
             romfs = 0x7275,
             rootfs = 0x53464846,
             rpc_pipefs = 0x67596969,
-            samba = 0x517B,
+            // samba = 0x517B, // same as smb
             sdcardfs = 0x5DCA2DF5,
             securityfs = 0x73636673,
-            selinux = 0xF97CFF8C,
-            sffs = 0x786F4256, // same as vboxfs
-            sharefs = 0x01021994, // same as tmpfs
+            selinuxfs = 0xF97CFF8C,
```

Comment:
All these changes (add/del `fs` suffix) seem to be worth including in the breaking change doc.